source/server/listener_manager_impl.cc

```diff
@@ -226,6 +227,16 @@ ListenerImpl::ListenerImpl(const envoy::api::v2::Listener& config, const std::st
     addListenSocketOptions(Network::SocketOptionFactory::buildIpPacketInfoOptions());
     // Needed to return receive buffer overflown indicator.
     addListenSocketOptions(Network::SocketOptionFactory::buildRxQueueOverFlowOptions());
+    std::string listener_name =
+        config.has_udp_listener_config() ? config.udp_listener_config().udp_listener_name() : "";
+    if (listener_name.empty()) {
+      listener_name = "raw_udp_listener";
```

Comment:
you could remove this if you replace the line above with
config.udp_listener_config().udp_listener_name() : "raw_udp_listener";
yeah?

Reply:
config.udp_listener_config().udp_listener_name() might also be empty. Every field is proto3 is optional...